01-data-structures/01-introduction-to-data-structures/line/line.rb

```diff
@@ -3,32 +3,45 @@
 # People may leave the line whenever they see fit and those behind them take their place.
 class Line
   attr_accessor :members
+  attr_accessor :line_length
 
   def initialize
-    self.members = []
+    @members = []
+    @line_length = @members.length
   end
 
-  def join(person)
+  def join person 
+    @members.push(person)
+    @line_length = @members.length
   end
 
-  def leave(person)
+  def leave person 
+    @members.delete(person)
+    @line_length = @members.length
```

Comment:
same comment as before if it's just a .length, then why have it be it's own thing?